src/doc/complement-design-faq.md

```diff
@@ -174,3 +174,10 @@ bindings.
 See also [a long thread][alt] on renaming `let mut` to `var`.
 
 [alt]: https://mail.mozilla.org/pipermail/rust-dev/2014-January/008319.html
+
+## Why no `--x` or `x++`?
+
+Pre- and post- increment, while convenient, are also fairly complex. They
+require knowledge of evaluation order, and often lead to subtle bugs and
+undefined behavior in C and C++. `x = x + 1` and `x = x - 1` is only slightly
```

Comment:
I’d go for `x += 1` and `x -= 1` here, because they’re both shorter and their meaning looks, at least to me, closer to the increment and decrement ops.